### DISCUSSION TITLE
[APPRECIATION] Nuxt team is awesome!

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [nuxt/nuxt#22230] [Kind words]
I was gonna open a question about if it was possible to retrieve a list of all Nuxt available modules.

But decided to try making a GET request to https://nuxt.com/modules and got an HTML response...

Was about to give up... then I tried with https://nuxt.com/api/modules.... and surprise!

![image](https://github.com/nuxt/nuxt/assets/10191085/3c2b538b-b666-417c-8f7f-1f59771cd5c8)

You guys take so many things into consideration! This ecosystem is solid! 

Devtools is getting better each day!

Huge props to all the team members. 

Lots of health, peace, and success. 💚🔥🤝

**Comment by Hebilicious** [comment]
There's also https://api.nuxt.com/modules !
